lib/database/message.php

```diff
@@ -2,10 +2,10 @@
 
 function CreateNewMessage($author, $destUser, $messageTitle, $messagePayloadIn)
 {
-    sanitize_sql_inputs($author, $destUser, $messageTitle, $messagePayloadIn);
-
     $messagePayload = nl2br($messagePayloadIn);
 
+    sanitize_sql_inputs($author, $destUser, $messageTitle, $messagePayloadIn);
```

Comment:
the important change here is actually moving the `messagePayload` line, so the <br> conversion is performed on a not-sanitized input.

Reply:
makes sense 👍